doc/releases/changelog-dev.md

```diff
@@ -47,6 +47,33 @@
 * New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
   [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
 
+* We can now perform QPE on quantum operations.
+
+  ```python3
+  dev = qml.device("default.qubit", wires=2)
+
+  unitary = [qml.RX(np.pi / 2, wires=[0]), qml.CNOT(wires=[0, 1])]
+  eigenvector = np.array([-1/2, -1/2, 1/2, 1/2])
+
+  n_estimation_wires = 5
+  estimation_wires = range(2, n_estimation_wires + 2)
+  target_wires = [0, 1]
+
+  dev = qml.device("default.qubit", wires=n_estimation_wires + 2)
+
+  @qml.qnode(dev)
+  def circuit():
+      qml.QubitStateVector(eigenvector, wires=target_wires)
+      QuantumPhaseEstimation(
+          unitary,
+          target_wires=target_wires,
+          estimation_wires=estimation_wires,
+      )
+      return qml.probs(estimation_wires)
+
+  phase_estimated = np.argmax(circuit()) / 2 ** n_estimation_wires
+  ```
```

Comment:
We don't need an example in the changelog for this case.  We want to restrict examples to big ticket items where we want to draw a lot of attention. Otherwise, people can just read docstrings.